index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie-edge">
+    <title>Project 1</title>
+
+    <link rel="stylesheet" href="./styles/normalize.css">
+    <link rel="stylesheet" href="./styles/base.css">
+    <link rel="stylesheet" href="./styles/index.css">
+</head>
+
+<body>
+
+    <header class="header">
+
+        <div class="top-bar">
+            <img class="top-bar__logo" src="./images/logo.svg" alt="Nova logo">
+            
+            <nav class="top-bar__menu menu">
+                <ul class="menu__list">
+                    <li class="menu__item"><a class="menu__link" href="">The Team</a></li>
+                    <li class="menu__item"><a class="menu__link" href="">Pricing</a></li>
+                    <li class="menu__item"><a class="menu__link" href="">Features</a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="top-container">
+            <section class="top-container__text-box text-box">  
+                <h1 class="text-box__title">Beautiful Free Nova template</h1>
+                <p class="text-box__paragraph">A top notch premium quality template for your next web project.</p>
+                <div class="text-box__download-button"><a class="text-box__download-link" href="">Download FREE!</a></div>
+            </section>
+
+            <aside class="top-container__img-box">
+                <img class="top-container__img" src="./images/screen.png" alt="App screen">
+            </aside>
+        </div>
+            
+    </header>
+
+    <section class="container--black">
+
+        <h2 class="container__title">FEATURES</h2>    
+        
+        <div class="container__container">
+            <article class="container__article article">
```

Comment:
Można też użyć `ul > li` jako lista cech. Osobiście wszędzie gdzie mamy "listę elementów" to właśnie używam `ul > li`